### PR TITLE
Always open ungit at /content/datalab/notebooks.

### PR DESCRIPTION
**Pull request by jimmc**
Fix for #1206 
This assumes a constant path for the repo location; it does not pull 'datalab' from the current location.

### CODE REVIEW

File: sources/web/datalab/static/datalab.js [hunk 1]
```diff
@@ -292,16 +292,8 @@ function initializePage(dialog, saveFn) {
...
-      // Extract the current directory name
-      if (path.indexOf('/tree') == 0) {
-        path = path.substr('/tree'.length);
-      } else if (path.indexOf('/notebooks') == 0) {
-        path = path.substr('/notebooks'.length);
-        path = path.substr(0, path.lastIndexOf('/'));
-      }
-      path = '/content' + path;
+      // Always open at the root of the notebooks repository
+      path = '/content/datalab/notebooks';
```
Comment (ojarjur) [inline sources/web/datalab/static/datalab.js]: This should still include "var " to declar the `path` variable.

File: sources/web/datalab/static/datalab.js [hunk 2]
```diff
@@ -292,16 +292,8 @@ function initializePage(dialog, saveFn) {
...
       prefix = window.location.protocol + '//' + window.location.host;
```
Comment (ojarjur) [inline sources/web/datalab/static/datalab.js]: This should have a "var" to declare the variable.

Comment (jimmc) [inline sources/web/datalab/static/datalab.js]: Fixed.

File: sources/web/datalab/static/datalab.js (hunk 1 above)
Comment (jimmc) [inline sources/web/datalab/static/datalab.js]: Fixed.

**Pull request merged**